newscoop/template_engine/metaclasses/MetaLanguage.php

```diff
@@ -84,4 +85,4 @@ public static function createFromCode($code)
             $languageId = Language::GetLanguageIdByCode($code);
             return new self($languageId);
         }
-    }
\ No newline at end of file
+    }
```

Comment:
hmm weird, something wrong is with my editor, it shows me there is no new line, ignore my comment for now